client/java/src/main/java/io/openlineage/client/transports/FacetsConfig.java

```diff
@@ -21,4 +21,8 @@ public class FacetsConfig {
   @Getter
   @JsonProperty("disabled")
   private String[] disabledFacets;
+
+  @Getter
+  @JsonProperty("custom_environment_variables")
```

Comment:
@Anirudh181001 I know it's a kind of a late remark, but would you be able to include description of this parameter in Spark integration docs: https://github.com/OpenLineage/OpenLineage/tree/main/integration/spark#general  ?